packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -176,6 +177,25 @@ devServerFeature.setup(
                 createConfigMiddleware(overrideConfig),
             ]);
 
+            app.get('/feature-graph', (req, res) => {
+                const featureName = req.query['feature-name'] as string;
+
+                const visitedFeatures = {} as { [propName: string]: number };
+
+                const links = buildFeatureLinks(features.get(featureName)!.exportedFeature, visitedFeatures, 0);
+
+                const graph = {
+                    nodes: Object.keys(visitedFeatures)
+                        .map((name) => ({ name, id: name, group: visitedFeatures[name] }))
+                        .concat({
+                            name: features.get(featureName)!.exportedFeature.id,
+                            id: features.get(featureName)!.exportedFeature.id,
+                            group: 0,
```

Comment:
wouldn't the group be always 0?

Reply:
Added a doc, It's the root feature, and it's always 0